mne/source_estimate.py

```diff
@@ -2995,6 +3004,8 @@ def _prepare_label_extraction(stc, labels, src, mode, allow_empty, use_sparse):
         if len(this_vertidx) == 0:
             bad_labels.append(label.name)
             this_vertidx = None  # to later check if label is empty
+        elif mode is None:  # no flip if no source activation provided
+            this_flip = []
```

Comment:
I wouldn't add this. Instead, wherever you need this function, pass `mode='mean'` and `this_flip` will be `None`

Reply:
It now says

mne/source_estimate.py#L2957
Added line #L2957 was not covered by tests

Should I add a function test_pepare_label_extraction to test_source_estimate.py?

Reply:
No need, you'll hit this one line in your next PR -- we can wait for that :)